addons/project/models/project.py

```diff
@@ -2160,12 +2160,18 @@ def _task_message_auto_subscribe_notify(self, users_per_task):
                 values.update(assignee_name=user.sudo().name)
                 assignation_msg = self.env['ir.qweb']._render('project.project_message_user_assigned', values, minimal_qcontext=True)
                 assignation_msg = self.env['mail.render.mixin']._replace_local_links(assignation_msg)
+                subtitles = [task.display_name]
+                if task.date_deadline:
```

Comment:
You have code for date_deadline here, and some generic above. I think it is better to handle deadline per model instead of the generic code in auto subscribe. Unless there are some issues with auto subscribe notification ?

Reply:
Thanks, same as above.